### ISSUE TITLE
Scroll Refresh/Crashing on Mobile

### ISSUE THREAD

**Issue by cvanem** [BIDMCDigitalPsychiatry/AppMapDB#104]
**Issues:**

iphone 11 pro:  Works periodically but then crashed with the "A problem repeatedly occurred on http://...".  Also, in the middle of use it will auto-refresh. 

**Notes:**
The auto-refresh bug is probably caused by the incremental loading of the data, which was introduced to increase load times.  The code loads 100 rows at a time.

**TODO**
Needs further investigation as to what is causing the issue.

**Comment by ecamach13** [comment]
Uploading RPReplay_Final1642536602.MP4…

**Comment by ecamach13** [comment]
https://user-images.githubusercontent.com/70787908/150012291-45f49c28-ea1a-477a-8f52-42e4e9010193.MP4

**Comment by ecamach13** [comment]
Above is a video of the crashing that I'm seeing on my phone at this moment ^
Even when it goes to a white screen, it doesn't return where you left it

**Comment by ecamach13** [comment]
Below are some emails I have received from individuals navigating the app on their phones. 

"I see how what to do, but when I am trying to do anything I keep getting pop up error messages. I have to keep reloading and it won’t let me go any further." - they confirmed they were experiencing this on their phone 

"I can't find the take survey button and the page keeps freezing, both on my phone and on the computer."

"Thank you so much. I’m looking forward to this. However, the web page keeps crashing. Is it on my end or are you having trouble?" - they confirmed they were experiencing this on their iPhone 

"I keep trying to search through the options on the mindapps page, however, it kept crashing…. 
I’m using chrome and on my phone."

**Comment by cvanem** [comment]
@ecamach13 
Thanks for the additional information.  I am still un-able to reproduce this on my end and have tried many different scenarios, multiple phones and multiple different computers.  It is hard for me to fix something that I can't reproduce and no error message is given.  My best guess is that users may be experiencing some mobile performance issues or device limitations, but I have not been able to confirm. 

**I just pushed an update that simplifies the data load and should help with performance and re-rendering of the table.  Can you please refresh the app on your phone and try to see if you can still reproduce the error?**  I am also working on a new grid view for the list that should also help with performance when completed.

**Any idea why users are seeing this error now and we did not see it previously?  Are there more users that are using the website now then there were before or did something change that started triggering this error?**

**Comment by ecamach13** [comment]
I just tried navigating the mobile version a couple of times and it has not frozen or crashed yet, so I think the push may have fixed it! 

The first time we heard of the crashing was when someone contacted John on 1/5 when I emailed you. This was before the start of the study with the app surveys. The reason there's more reports on it is because there's more people using the website to navigate to the surveys. I don't think the crashing is related to the users, because the crashes started before the additional users began using the website.
